micro-frontends/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "openmrs-mfe-host",
+  "version": "0.0.1",
+  "description": "Hosting microfrontends for openmrs",
+  "main": "dist/index.js",
+  "author": "bahmni@thoughtworks.com",
+  "license": "AGPL-3.0",
```

Comment:
@AshishkurianTw openmrs-module-bahmniapps is licensed under Apache-2.0. Should micro-frontends/ also adhere to the same license agreement, or is it acceptable to maintain AGPL-3.0 license?

cc: @angshu, @binduak